src/Invictus.Testing/LogicAppException.cs

```diff
@@ -0,0 +1,127 @@
+﻿using System;
+using System.Runtime.Serialization;
+using System.Security.Permissions;
+using GuardNet;
+
+namespace Invictus.Testing
+{
+    /// <summary>
+    /// Thrown when a problem occurs during interaction with a logic app running in Azure.
+    /// </summary>
+    [Serializable]
+    public class LogicAppException : Exception
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="LogicAppException"/> class.
+        /// </summary>
+        public LogicAppException()
+        {
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="LogicAppException"/> class.
+        /// </summary>
+        /// <param name="message">The message that describes the exception.</param>
+        public LogicAppException(string message) : base(message)
+        {
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="LogicAppException"/> class.
+        /// </summary>
+        /// <param name="message">The message that describes the exception.</param>
+        /// <param name="innerException">The exception that is the cause of the current exception</param>
+        public LogicAppException(string message, Exception innerException) : base(message, innerException)
```

Comment:
I would remove this overload, we need to have logic app information as a minimum

Reply:
See ⏫ .